src/patients/care-goals/CareGoalForm.tsx

```diff
@@ -87,50 +85,68 @@ const CareGoalForm = (props: Props) => {
-          />
+          <div className="form-group" data-testid="prioritySelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.priority.label')}
+              htmlFor={t('patient.careGoal.priority.label')}
+              isRequired
+            />
+            <Select
+              id="prioritySelect"
+              options={priorityOptions}
+              onChange={onPriorityChange}
+              defaultSelected={priorityOptions.filter(({ value }) => value === priority)}
+              isInvalid={!!careGoalError?.priority}
+              disabled={disabled}
+            />
+          </div>
         </Column>
       </Row>
       <Row>
         <Column sm={6}>
-          <SelectWithLabelFormGroup
-            name="status"
-            label={t('patient.careGoal.status')}
-            isRequired
-            options={statusOptions}
-            defaultSelected={statusOptions.filter(({ value }) => value === status)}
-            isEditable={!disabled}
-            isInvalid={!!careGoalError?.status}
-            onChange={(values) => {
-              onFieldChange('status', values[0])
-              setStatus(values[0] as CareGoalStatus)
-            }}
-          />
+          <div className="form-group" data-testid="statusSelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.status')}
+              htmlFor={t('patient.careGoal.status')}
+              isRequired
+            />
+            <Select
+              id="statusSelect"
+              options={statusOptions}
+              onChange={(values) => {
+                onFieldChange('status', values[0])
+                setStatus(values[0] as CareGoalStatus)
+              }}
+              defaultSelected={statusOptions.filter(({ value }) => value === status)}
+              isInvalid={!!careGoalError?.status}
+              disabled={disabled}
+            />
+          </div>
         </Column>
         <Column sm={6}>
-          <SelectWithLabelFormGroup
-            name="achievementStatus"
-            label={t('patient.careGoal.achievementStatus')}
-            isRequired
-            options={achievementsStatusOptions}
-            defaultSelected={achievementsStatusOptions.filter(
-              ({ value }) => value === achievementStatus,
-            )}
-            isEditable={!disabled}
-            isInvalid={!!careGoalError?.achievementStatus}
-            onChange={(values) => {
-              onFieldChange('achievementStatus', values[0])
-              setAchievementStatus(values[0] as CareGoalAchievementStatus)
-            }}
-          />
+          <div className="form-group" data-testid="achievementStatusSelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.achievementStatus')}
+              htmlFor={t('patient.careGoal.achievementStatus')}
```

Comment:
See some of the above comments for `htmlFor`. I might be misinterpreting how to use the attribute.